Improve WSL2 Installation experience with a PowerShell script [skip ci] (#4150)
drud/ddev: docs/content/users/install/ddev-installation.md

    **WSL2 is the recommended installation method for all Windows users**.

>      **Using WSL2 to install and run DDEV is not the same as using Docker Desktop's WSL2 engine, which itself runs in WSL2, but can serve applications running in both traditional WIndows and inside WSL2.**

Author: ```suggestion
    **Using WSL2 to install and run DDEV is not the same as using Docker Desktop's WSL2 engine, which itself runs in WSL2, but can serve applications running in both traditional WIndows and inside WSL2.**
```

    * Installing or upgrading to the latest Docker Desktop for Windows with WSL2 enabled.
    * Installing DDEV inside your distro.

    ### WSL2 Scripted Installation

    You can do these things manually, or you can do most of it with the provided PowerShell script. 
    In all cases:
    
    1. Install WSL2 with an Ubuntu distro. On a system without WSL2, just run:
        ```powershell
        wsl --install
        ```

>         If that doesn't work for you, see the [manual installation](https://docs.microsoft.com/en-us/windows/wsl/install-manual) and linked [troubleshooting](https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#installation-issues).

Author: ```suggestion
        If that doesn't work for you, see the [manual installation](https://docs.microsoft.com/en-us/windows/wsl/install-manual) and linked [troubleshooting](https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#installation-issues).
 ```

     2. Use `wsl -l -v` to see your distros and which is the default.
 If you don't already have an Ubuntu distro with it set as default, install it with `wsl --install Ubuntu`.

>     2. Install Docker Desktop and start it. If you already have chocolatey, `choco install -y docker-desktop` or [download Docker Desktop from Docker](https://www.docker.com/products/docker-desktop/).

Author: ```suggestion
    2. Install Docker Desktop. If you already have chocolatey, `choco install -y docker-desktop` or [download Docker Desktop from Docker](https://www.docker.com/products/docker-desktop/).
    3. Start Docker Desktop.
```
